src/scripts/ccget

```diff
@@ -179,7 +181,15 @@ def ccget():
                         continue
                 else:
                     if hasattr(data, attr):
-                        print("%s:\n%s" % (attr, getattr(data, attr)))
+                        print(attr)
+                        attr_val = getattr(data, attr)
+                        # List of attributes to be printed with new lines
+                        if attr in ['mocoeffs', 'moenergies', 'moments', 
```

Comment:
Would it be possible to get this list from [here](https://github.com/cclib/cclib/blob/master/src/cclib/parser/data.py#L162)?